BACnetClient.cs

```diff
@@ -1281,8 +1278,9 @@ public IList<BacnetReadAccessResult> EndReadPropertyMultipleRequest(BacnetAsyncR
             {
                 return request.GetResult(Timeout, Retries, r =>
                 {
-                    var byteCount = Services.DecodeReadPropertyMultipleAcknowledge(
-                        r.Result, 0, r.Result.Length, out var values);
+                    var byteCount =
+                        Services.DecodeReadPropertyMultipleAcknowledge(r.Addr, r.Result, 0, r.Result.Length,
+                            out var values);

```

Comment:
This is Visual Studio or ReSharper breaking the line for me, I usually don't do this myself. What's your setting there?

Reply:
To be honest I always correct it myself if I get such effect 🙈 I don't know if this is possible to set to achieve this effect automaticaly. Do see this version as more readable or not?

Reply:
I dislike both and would prefer to scroll 😉 
Now that I took a look at the available ReSharper-Settings, I'd prefer "Chop if long or multiline" with "prefer wrap after (", which looks like this:

```
CallMethod(
    arg1,
    arg2,
    arg3,
    arg4,
    arg5);
```
but really... any setting is fine, as long as ReSharper takes care of it.

Reply:
Good thing this repo doesn't have dozens of contributors as I can already see an ideaology discussion here 😆 Since we have way bigger problems in this codebase than style, how about this. For now please stick to my suggestions regarding code style, and I promise in the future to have ReSharper styles files in the repo and stick to it's automatic formatting? Maybe in time more people will join and we can use democracy to decide (or we have a big argument and everyone forks the repo 😝)